package/src/components/AttachmentPicker/AttachmentPicker.tsx

```diff
@@ -326,35 +337,24 @@ export const AttachmentPicker = React.forwardRef(
           : statusBarHeight
         : 0;
 
-    const initialSnapPoint = useMemo(
-      () =>
-        attachmentPickerBottomSheetHeight ?? Platform.OS === 'android'
-          ? 308 +
-            (fullScreenHeight - screenHeight + androidBottomBarHeightAdjustment) -
-            handleHeight
-          : 308 + (fullScreenHeight - screenHeight + androidBottomBarHeightAdjustment),
-      [
-        attachmentPickerBottomSheetHeight,
-        androidBottomBarHeightAdjustment,
-        fullScreenHeight,
-        handleHeight,
-        screenHeight,
-      ],
-    );
+    const initialSnapPoint =
+      attachmentPickerBottomSheetHeight ?? Platform.OS === 'android'
+        ? 308 + (fullScreenHeight - screenHeight + androidBottomBarHeightAdjustment) - handleHeight
+        : 308 + (fullScreenHeight - screenHeight + androidBottomBarHeightAdjustment);
```

Comment:
308 I think is the height of a 3x3 grid of images in the image picker (right, @vishalnarkhede?) - should we just make a variable for it so that bit of trivia doesn't get lost to time?

Not a blocker for this PR of course, just figured I'd mention it before I forget it

Reply:
I didn't know, so couldn't pick a variable name. @vishalnarkhede if you can give me a relevant variable name, will change it. Would prefer to do it in this PR.

Reply:
@khushal87 any suggestions?

Reply:
honestly I don't know what 308 is exactly.

Reply:
So I will let @santhoshvai do the needful here

Reply:
Even, I am not sure @santhoshvai

Reply:
308 is the default height of the bottom sheet 

<img width="1035" alt="image" src="https://user-images.githubusercontent.com/3846977/169787902-cdf3626e-530e-4ccb-8abe-0406e7889799.png">